projects/bp-gallery/src/helpers/growthbook.ts

```diff
@@ -11,6 +11,7 @@ const growthbookClientKey = import.meta.env.VITE_REACT_APP_GROWTHBOOK_CLIENTKEY;
 export type AppFeatures = {
   test_button: boolean;
   dummy_experiment: boolean;
+  paypal_mainpage: JSON;
```

Comment:
We wanted to use json since we want to enable/disable the whole donation function. If we split this we'd have two feature flags that depend on each other, so I think it's better to have it as Json

Reply:
That code wouldn't split the feature, it just says which type the _one_ feature `paypal_mainpage` has. And that type just mirrors the JSON structure you defined in the Growthbook UI. And again, `JSON` doesn't mean "any JSON data" but rather this type:
```ts
interface JSON {
    /**
     * Converts a JavaScript Object Notation (JSON) string into an object.
     * @param text A valid JSON string.
     * @param reviver A function that transforms the results. This function is called for each member of the object.
     * If a member contains nested objects, the nested objects are transformed before the parent object is.
     */
    parse(text: string, reviver?: (this: any, key: string, value: any) => any): any;
    /**
     * Converts a JavaScript value to a JavaScript Object Notation (JSON) string.
     * @param value A JavaScript value, usually an object or array, to be converted.
     * @param replacer A function that transforms the results.
     * @param space Adds indentation, white space, and line break characters to the return-value JSON text to make it easier to read.
     */
    stringify(value: any, replacer?: (this: any, key: string, value: any) => any, space?: string | number): string;
    /**
     * Converts a JavaScript value to a JavaScript Object Notation (JSON) string.
     * @param value A JavaScript value, usually an object or array, to be converted.
     * @param replacer An array of strings and numbers that acts as an approved list for selecting the object properties that will be stringified.
     * @param space Adds indentation, white space, and line break characters to the return-value JSON text to make it easier to read.
     */
    stringify(value: any, replacer?: (number | string)[] | null, space?: string | number): string;
}
```
So if you really want to say "some JSON data", then you need to use the one from `@growthbook/growthbook`:
```ts
export type JSONValue =
  | null
  | number
  | string
  | boolean
  | Array<JSONValue>
  | { [key: string]: JSONValue };
```
This type actually represents "some JSON data". But I encourage you not to use this type as it unecessarily underspecifies the type you actually use, which is this one:
```ts
type PaypalMainpageFeatureValueType = {
  clientId: string;
  donationText: string;
};
```